CHANGELOG.md

```diff
@@ -1,146 +1,113 @@
-that they can continue to be used that way for less line noise.
+Additionally, this means that `SignalProducer` operators are less “magic”. In RAC 3 the `Signal` operators were implicitly lifted to work on `SignalProducer` via `|>`. This was a point of confusion for some, especially when browsing the
+source looking for these operators. Now as protocol extensions, the `SignalProducer` operators are explicitly implemented in terms of their `Signal` counterpart when available.
 
-#### Renamed signal generic parameters
+#### Removal of `|>` custom operator
 
-The generic paramters of `Signal`, `SignalProducer`, and other related types
-have been renamed to `Value` and `Error` from `T` and `E` respectively. This
-is in-line with changes to the standard library to give more descriptive names
-to type parameters for increased clarity. This should have limited impact,
-only affecting generic, custom signal/producer extensions.
+As already alluded to above, the custom `|>` operator for chaining signals has been removed. Instead standard method calling syntax is used for chaining operators.
 
-#### Added flatten to all signal and producer combos
+#### Event cases are no longer boxed
 
-Before this change, `flatten` and `flatMap` were only defined for
-signals-of-producers and producers-of-producers. After much discussion it
-was determined these should be safe for the remaining combinations of signals
-and producers. See #2449
+The improvements to associated enum values in `Swift 2` mean that `Event` case no longer need to be `Box`ed. In fact, the `Box` dependency has been removed completely from `RAC 4`.
 
-#### Added missing SignalProducer operators
+#### Replacements for the `start` and `observer` overloads
 
-There were some `Signal` operators that were missing `SignalProducer`
-equivalents:
+The `observe` and `start` overloads taking `next`, `error`, etc. optional function parameters have been removed. They’ve been replaced with methods taking a single function with
+the target `Event` case — `observeNext`, `startWithNext`, and the same for `failed` and `completed`. See #2311 and #2318 for more details.
 
-* `takeUntil`
-* `combineLatestWith`
-* `sampleOn`
-* `takeUntilReplacement`
-* `zipWith`
+#### Renamed `try` and `catch` operators
 
-#### Renamed PropertyOf<T> to AnyProperty<T>
+The `try` and `catch` operators were renamed because of the addition of the error handling keywords with the same name. They are now `attempt` and `flatMapError` respectively. Also, `tryMap` was renamed to `attemptMap` for consistency.
 
-This is in-line with changes to the standard library in Swift 2.
+#### `flatten` and `flatMap` are now possible for all 4 combinations of `Signal`+`SignalProducer`
 
-#### Publicized Bag and Atomic
+This fills a gap that was missing in `RAC 3`. It’s a common pattern to have a hot `Signal` of values that need to be mapped to “work” — `SignalProducer`, or a `Signal` of `Signal`s.
```

Comment:
This reads a little awkwardly to me, it took me rereading it a few times to get what it was trying to say,